girder/lib/mongodb_proxy.py

```diff
@@ -0,0 +1,145 @@
+"""
```

Comment:
Sounds good.  I think I like 'external' best of those choices, as it is explicit.